dev/remote-linux-arm64-test.sh

```
@@ -0,0 +1,70 @@
+: "${REMOTE_DIR:?Set REMOTE_DIR to an absolute path on \$REMOTE used as the work directory}"
+
+FILTER=${1:-}
+
+# Run gradle inside the project's nix dev shell so OPENSSL_INCLUDE_DIR (and the rest of
+# the build env) is exported for cinterop, no matter what shell launched this script.
+nix develop --command gradle \
+  :integration-tests:linkDebugTestLinuxArm64 \
+  :test-harness:nats-server-daemon:installDist \
+  :test-harness:tls-test-server:installDist
+
+ssh "$REMOTE" "mkdir -p $REMOTE_DIR/test-bin $REMOTE_DIR/harness $REMOTE_DIR/logs"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)